docs/en/seatunnel-engine/rest-api.md

```diff
@@ -0,0 +1,121 @@
+---
+
+sidebar_position: 7
+-------------------
+
+# REST API
+
+Seatunnel has a monitoring API that can be used to query status and statistics of running jobs, as well as recent
+completed jobs. The monitoring API is a REST-ful API that accepts HTTP requests and responds with JSON data.
+
+## Overview
+
+The monitoring API is backed by a web server that runs as part of the node, each node member can provide rest api capability.
+By default, this server listens at port 5801, which can be configured in hazelcast.yaml like :
+
+```yaml
+network:
+    rest-api:
+      enabled: true
+      endpoint-groups:
+        CLUSTER_WRITE:
+          enabled: true
+        DATA:
+          enabled: true
+    join:
+      tcp-ip:
+        enabled: true
+        member-list:
+          - localhost
+    port:
+      auto-increment: true
+      port-count: 100
+      port: 5801
+```
+
+## API reference
+
+### Returns an overview over all jobs and their current state.
+
+<details>
+ <summary><code>GET</code> <code><b>/hazelcast/rest/maps/running-jobs</b></code> <code>(Returns an overview over all jobs and their current state.)</code></summary>
+
+#### Parameters
+
+```json
+{}
```

Comment:
If requestbody is empty, should remove it.

Reply:
I agree with you.